Tests/SentryTests/Integrations/Session/SentrySessionTrackerTests.swift

```diff
@@ -361,7 +361,21 @@ class SentrySessionTrackerTests: XCTestCase {
         
         assertNotificationNames(notificationNames)
     }
-    
+
+    func testForegroundBeforeStart() {
+        // -- Arrange --
+        goToForeground()
```

Comment:
@philipphofmann should we just mock the `SentryDependencyContainer.sharedInstance.application` here and create an UI test with a real app running?

Reply:
I don't think we need a UI test with a real app for this. What would be the benefit? I would simply pass in the `SentryUIApplication` as a dependency and then use a mock implementation as in SentryUIViewControllerSwizzlingTests for example. We could also extract a common `SentryUIApplication` for testing from `SentryUIViewControllerSwizzlingTests` or use a local one in here. Up to you.